serialization/src/main/scala/sbt/serialization/UTF8FileOutput.scala

```diff
@@ -0,0 +1,24 @@
+package sbt.serialization
+
+import java.io.{ File, FileOutputStream, BufferedWriter, OutputStreamWriter }
+import scala.pickling.Output
+import java.nio.charset.Charset
+
+class UFT8FileOutput(file: File) extends Output[String] {
+  private[this] val charset = Charset.forName("UTF-8")
```

Comment:
Maybe this should be in the companion object, so it's not looked up everytime we create this file, only when the class is loaded.